apps/meteor/client/views/room/hooks/useAppsContextualBar.ts

```diff
@@ -24,17 +23,12 @@ export const useAppsContextualBar = (): AppsContextualBarData | undefined => {
 	const viewId = params?.context;
 
 	useEffect(() => {
-		async function getAppData(appId: string): Promise<void> {
-			const app = await Apps.getApp(appId);
-			setAppInfo(app);
-		}
-
 		if (viewId) {
 			setPayload(getUserInteractionPayloadByViewId(viewId) as IUIKitContextualBarInteraction);
 		}
 
 		if (payload?.appId) {
-			getAppData(payload.appId);
+			setAppInfo({ id: payload.appId } as any);
```

Comment:
we should avoid any!! 
as you're not going to hold all app data you should fix the `appInfo` type